### PR TITLE
Feature/table paging

### PR DESCRIPTION
**Pull request by thomasschneiter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@lambda-it/parlamentsdienste-component-library",
-    "version": "1.7.3",
+    "version": "1.7.3-1",
```
Comment (matthiasbaldi) [inline package.json:3]: I think this here is an old thing. It should be .4 or even 1.8

File: src/components/pd-dropdown/pd-dropdown.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ export class Dropdown {
 
     @Listen('click', { target: 'body' })
     protected handleClick(ev: MouseEvent) {
-        if (closestElement('pd-dropdown', ev.target as HTMLElement) !== this.element) {
+        if (closestElement('pd-dropdown', ev.composedPath()[0] as HTMLElement) !== this.element) {
```
Comment (matthiasbaldi) [inline src/components/pd-dropdown/pd-dropdown.tsx:91]: Is it possible to get here an empty array or undefined or is an null check required?
But I think there should at least one item, even if it is only the host element.

File: package.json (hunk 1 above)
Comment (thomasschneiter) [reply package.json:3]: The idea is to give a pre-release version with table paging for testing the functionality

File: src/components/pd-dropdown/pd-dropdown.tsx (hunk 2 above)
Comment (thomasschneiter) [reply src/components/pd-dropdown/pd-dropdown.tsx:91]: composedPath()[0] always returns an element. Worst case the 'document' element

**Pull request merged**